modules/platform/nuxeo-platform-web-common/src/test/java/org/nuxeo/ecm/platform/ui/web/auth/TestNuxeoAuthenticationFilter.java

```diff
@@ -670,6 +672,41 @@ public void testAuthPluginFormRedirectToLoginPage() throws Exception {
         String entity = out.toString(UTF_8);
         assertTrue(entity, entity.contains(
                 "window.location = 'http://localhost:8080/nuxeo/dummy_login.jsp?requestedUrl=mystart%2Ffoo';"));
+
+        // check that start url was saved
+        assertEquals("mystart/foo", sessionAttributes.get(START_PAGE_SAVE_KEY));
+    }
+
+    /**
+     * Redirect to plugin login page without saving start URL.
+     */
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.web.common.test:OSGI-INF/test-authchain-dummy-no-start-url.xml")
+    public void testNoStartURLSaving() throws Exception {
+        HttpServletRequest request = mock(HttpServletRequest.class);
+        HttpServletResponse response = mock(HttpServletResponse.class);
+        HttpSession session = mock(HttpSession.class);
+        Map<String, Object> sessionAttributes = mockSessionAttributes(session);
+        when(request.getSession(anyBoolean())).thenReturn(session);
+        // mystart/ is defined as a start url in the XML config
+        mockRequestURI(request, "/mystart/foo", "", "");
```

Comment:
I guess you could call the 2-parameter method.

Reply:
👍  will do